.github/workflows/test.yml

```
@@ -11,6 +11,10 @@ jobs:
   test:
     name: Run Tests
     runs-on: ubuntu-latest
+spring:
```

actionlint (1.7.12)

[error] unexpected key "spring" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)